indymeet/urls.py

```diff
@@ -15,6 +15,7 @@
     path("search/", search_views.search, name="search"),
     # path("home/", home_views.test_page, name="test"),
     path("accounts/", include("accounts.urls")),
+    path("", include("home.urls")),
 ]
```

Comment:
@dawnwages  I know you used "" for the pupet URLS but I think we should reserve that space for our home.urls and , if I'm not mistaken ...pupet is a blog? So maybe we could have something more descriptive there like "blog/" ..?

Reply:
sorry, I'm just realizing why you said this. #41 has already been merged. You can add it here in this PR. Thanks, you're totally right.

Reply:
and I see addressing this also resolves your conflict. I'm down for this changed and can be merged after this. 👍🏾